server/controllers/plantsControllers.js

```diff
@@ -0,0 +1,95 @@
+import { PrismaClient } from '../generated/prisma/client.js';
+import storage from '../supabase/storageClient.js';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
```suggestion
import { randomUUID } from 'node:crypto';
```
we don't need an external lib for uuid, can just use the crypto module